feature/experimental/policy/ate_tests/base_vrf_selection_test.go

```diff
@@ -0,0 +1,465 @@
+		singletag.VlanId = ygot.Uint16(vlan)
+	}
+	//Add IPv4 stack
+	s4 := s.GetOrCreateIpv4()
+	if *deviations.InterfaceEnabled {
+		s4.Enabled = ygot.Bool(true)
+	}
+	a := s4.GetOrCreateAddress(me.IPv4)
+	a.PrefixLength = ygot.Uint8(plen4)
+
+	//Add IPv6 stack
+	s6 := s.GetOrCreateIpv6()
+	if *deviations.InterfaceEnabled {
+		s6.Enabled = ygot.Bool(true)
+	}
+	s6.GetOrCreateAddress(me.IPv6).PrefixLength = ygot.Uint8(plen6)
+
+	return i
+}
+
+// Configure Network instance on the DUT
+func configNetworkInstance(name string, peer *attrs.Attributes) *telemetry.NetworkInstance {
+
+	d := &telemetry.Device{}
+	ni := d.GetOrCreateNetworkInstance(name)
+
+	ni.Type = telemetry.NetworkInstanceTypes_NETWORK_INSTANCE_TYPE_L2L3
+	static := ni.GetOrCreateProtocol(telemetry.PolicyTypes_INSTALL_PROTOCOL_TYPE_STATIC, "STATIC")
+	ipv4Nh := static.GetOrCreateStatic("0.0.0.0/0").GetOrCreateNextHop(peer.IPv4)
+	ipv6Nh := static.GetOrCreateStatic("::/0").GetOrCreateNextHop(peer.IPv6)
+	ipv4Nh.Recurse = ygot.Bool(true)
+	ipv6Nh.Recurse = ygot.Bool(true)
+
+	return ni
+}
+
+func configForwardingPolicy(name string, matchType string, ate *attrs.Attributes) *telemetry.NetworkInstance_PolicyForwarding {
+
+	d := &telemetry.Device{}
+	ni := d.GetOrCreateNetworkInstance(name)
+	ipv4Address := "0.0.0.0/0"
+	ipv6Address := "::/0"
+	//TODO dscpset := []uint8{42, 46}
+
+	//Match policy
+	policyFwding := ni.GetOrCreatePolicyForwarding()
+
+	fwdPolicy1 := policyFwding.GetOrCreatePolicy("Match-IpipDscp4246")
+	fwdPolicy1.GetOrCreateRule(1).GetOrCreateIpv4().Dscp = ygot.Uint8(46)
+	//TODO fwdPolicy1.GetOrCreateRule(1).GetOrCreateIpv4().DscpSet = dscpset
+	fwdPolicy1.GetOrCreateRule(1).GetOrCreateIpv4().Protocol = telemetry.PacketMatchTypes_IP_PROTOCOL_IP_IN_IP
+	fwdPolicy1.GetOrCreateRule(1).GetOrCreateAction().NetworkInstance = ygot.String("10")
+	fwdPolicy1.GetOrCreateRule(2).GetOrCreateIpv4().Dscp = ygot.Uint8(42)
+	fwdPolicy1.GetOrCreateRule(2).GetOrCreateIpv4().Protocol = telemetry.PacketMatchTypes_IP_PROTOCOL_IP_IN_IP
+	fwdPolicy1.GetOrCreateRule(2).GetOrCreateAction().NetworkInstance = ygot.String("10")
+	fwdPolicy1.GetOrCreateRule(3).GetOrCreateIpv4().SourceAddress = ygot.String(ipv4Address)
+	fwdPolicy1.GetOrCreateRule(3).GetOrCreateAction().Discard = ygot.Bool(true)
+	fwdPolicy1.GetOrCreateRule(1).GetOrCreateIpv6().Dscp = ygot.Uint8(46)
+	//TODO fwdPolicy1.GetOrCreateRule(1).GetOrCreateIpv6().DscpSet = dscpset
+	fwdPolicy1.GetOrCreateRule(1).GetOrCreateIpv6().Protocol = telemetry.PacketMatchTypes_IP_PROTOCOL_IP_IN_IP
+	fwdPolicy1.GetOrCreateRule(1).GetOrCreateAction().NetworkInstance = ygot.String("10")
+	fwdPolicy1.GetOrCreateRule(2).GetOrCreateIpv6().Dscp = ygot.Uint8(42)
+	fwdPolicy1.GetOrCreateRule(2).GetOrCreateIpv6().Protocol = telemetry.PacketMatchTypes_IP_PROTOCOL_IP_IN_IP
+	fwdPolicy1.GetOrCreateRule(2).GetOrCreateAction().NetworkInstance = ygot.String("10")
+	fwdPolicy1.GetOrCreateRule(3).GetOrCreateIpv6().SourceAddress = ygot.String(ipv6Address)
```

Comment:
Can you help remove IPv6 as source here? (the README.md also suggests that it's not required). Thanks.